sympy/parsing/mathematica.py

```diff
@@ -23,6 +23,10 @@ def parse(s):
         lambda m: parse(m.group(1)) + translateFunction(
             m.group(2)) + parse(m.group(3))),
 
+        # Space multiplication between two functions
+        (r"\A(\w+\[[^\]]+[^\[]*\])( )(\w+\[[^\]]+[^\[]*\])\Z",
```

Comment:
I mean, many spaces, like: `Sin[x]    Cos[x]`. I think it's still a valid Mathematica expression.